esp/src/src/nls/es/hpcc.ts

```diff
@@ -169,9 +174,11 @@ export = {
     Critical: "Crítico",
     CrossTab: "Tabulación cruzada",
     CSV: "CSV",
+    CustomLogColumns: 'Columnas de Registro Personalizado',
     Dali: "Dali",
     DaliAdmin: "Administrador de Dalí",
     DaliIP: "IP de Dali",
+    DaliPromptConfirm: '¿Estás seguro? Las sumisiones de Dali Admin son permanentes y no se pueden deshacer.',
```

Comment:
This is fine as is, but this form is too intimate: ¿Estás seguro? I'd opt for ¿Está seguro?

Reply:
Changes made.